README.md

```diff
@@ -51,24 +51,25 @@ Here are a few other utility commands you may find useful:
 
 Sometimes you will want to use the latest version of the `web-design-standards` repo. Follow these steps to do so:
 
-1. Clone the latest version of the [`web-design-standards` repo](https://github.com/uswds/uswds/tree/develop).
-1. Run `npm install` to install the dependencies required for the package in the `web-design-standards` directory.
-1. Run `npm run build` to create the built version of USWDS in the `web-design-standards` directory.
-1. Run `npm link` in the _root level_ of the `web-design-standards` directory on your local machine.
-1. Run `npm link uswds` in the _root level_ of the `web-design-standards-docs` directory on your local machine.
-1. Set the `FRACTAL_BASE_URL` env var to the running fractal instance for `web-design-standards`. In your terminal window in the `web-design-standards-docs` directory, enter `export FRACTAL_BASE_URL="http://127.0.0.1:3000"`.
+1. Clone the latest version of the [`uswds` repo](https://github.com/uswds/uswds/tree/develop).
+1. Run `npm install` to install the dependencies required for the package in the `uswds` directory.
+1. Run `npm run build` to create the built version of USWDS in the `uswds` directory.
+1. Run `npm link` in the _root level_ of the `uswds` directory on your local machine.
+1. Run `npm link uswds` in the _root level_ of the `uswds-site` directory on your local machine.
+1. Run `npm start` in the `uswds` directory, and nake a note of the `Local URL` that Fractal is serving.
+1. In another terminal window, set the `FRACTAL_BASE_URL` env var to the running Fractal instance for `uswds`. In your terminal window in the `uswds-site` directory, enter `export FRACTAL_BASE_URL="http://127.0.0.1:3000"` (or the `Local URL` noted above).
+1. In the `uswds-site` window, run `npm run serve` in the `uswds-site` directory to start the Jekyll server.
 1. Run `npm run watch` in both project directories to have changes automatically built and compiled on changes to any asset files.
```

Comment:
We only need to run `npm run watch` in `uswds-site` as `npm start` already runs a watch in the `uswds` directory

Reply:
Ran through these steps on fresh clones and everything worked. Still newish, but if you save work on `uswds`, you still need to save a file on `uswds-site` in order to trigger the build and see the updated component, even if you didn't make changes to `uswds-site`.

Reply:
OK, updated to clarify this point